### ISSUE TITLE
Make it clearer that STEP 1 to select a query is required

### ISSUE THREAD

**Issue by TranslatorIssueCreator** [NCATSTranslator/Feedback#158]

**Comment by sierra-moxon** [comment]
@sstemann @Genomewide @gprice1129 @dnsmith124  - can you help me assign a milestone to this?  I think I heard Andy say that you were thinking of reworking the question-select box in the UI?

**Comment by dnsmith124** [comment]
That is correct, the entire process of query selection will almost certainly receive a reimplementation to directly address this issue. As it is now it's not very intuitive. At the very least we can set "What drugs may treat" as the default selection in the meantime, allowing the user to start typing right away.

**Comment by andrewsu** [comment]
I don't use the UI that often, so I consider myself a pseudo-naive user.  Despite hearing this issue being discussed on several calls already, I still was very confused when trying to execute a query.  I tested incognito mode, multiple browsers, and multiple computers, and then started prepping a slack thread before finally remembering the reason.  In addition to putting in a default selection (as @dnsmith124 suggests above), consider disabling the autocomplete box when a query mode has not been set.  It was very confusing that I was able to get my cursor in the text box, but it was not accepting keyboard input.

IMHO, making some fix here should be a high priority and should get implemented/released before Aug 18 (the current milestone).  I think this is one of those small, early roadblocks that could absolutely cause a less-motivated user to leave and dismiss the Translator tool/project entirely.
